tests/test_data.py

```diff
@@ -162,3 +146,37 @@
     "triggersIds": [],
     "alertId": 304255
 }
+
+alert_D = {
```

Comment:
I tried to use alert_D for this but it broke other unit tests because they call alert.to_dict() which doesnt store the objects, like triggers and notifications.

I think it's better to add alert_all_info_D for testing that the objects are in the alert after calling GET /alerts/allinfo